rtsf-at-checkout-cv-region-of-interest/src/enterexit.py

```diff
@@ -169,7 +169,8 @@ def create_pipelines():
             'destination': {
                 "type": "mqtt",
                 "host": mqttDestHost,
-                "topic": "AnalyticsData"
+                "topic": "AnalyticsData",
+                "timeout": 1000
```

Comment:
Is there a reason we are assigning timeout to its [default value](https://github.com/opencv/gst-video-analytics/blob/master/gst/elements/gvametapublish/mqttpublisher.c#L61) here (e.g., is it being overridden in some cases)?

Reply:
The pipeline templates originally set it to 1000 and I just decided to add it as a parameter here, moving it from the template to the request. It can be completely removed but figured a little more control is probably better